idp-default.properties

```diff
@@ -0,0 +1,264 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+# Follow properites have their values assigned via 'overrides' environment variables of config server docker.
+# DO NOT define these in any of the property files.  They must be passed as env variables.  Refer to config-server
+# helm chart:
+# db.dbuser.password
+# keycloak.external.url
+# keycloak.internal.host
+# keycloak.internal.url
+# keycloak.admin.password
+# mosip.auth.client.secret   (convention: <realm>.<keycloak client name>.secret)
+# mosip.ida.client.secret
+# mosip.admin.client.secret
+# mosip.reg.client.secret
+# mosip.prereg.client.secret
+# softhsm.kernel.pin
+# softhsm-security-pin
+# email.smtp.host
+# email.smtp.username
+# email.smtp.secret
+# mosip.kernel.tokenid.uin.salt
+# mosip.kernel.tokenid.partnercode.salt
+# mosip.api.internal.url
+# mosip.api.public.url
+
+## IdP MISP License key
+mosip.idp.misp.license.key=hVq9XOvlRokmkG23JL7wAJhEGM5ZzgO7Hnx3omJ1ghMap78nYh
+
+# Secure endpoints with required authority
+mosip.idp.security.auth.post-urls={'${server.servlet.path}/client-mgmt/**' : {'SCOPE_add_oidc_client'} , \
+  \ '${server.servlet.path}/system-info/**' : { 'SCOPE_upload_certificate'} }
+mosip.idp.security.auth.put-urls={'${server.servlet.path}/client-mgmt/**' : { 'SCOPE_update_oidc_client'} }
+mosip.idp.security.auth.get-urls={'${server.servlet.path}/system-info/**' : { 'SCOPE_get_certificate'} }
+
+# CSRF token check disabled endpoints
+mosip.idp.security.ignore-csrf-urls=${server.servlet.path}/oidc/**,${server.servlet.path}/oauth/**,\
+  ${server.servlet.path}/actuator/**,/favicon.ico,${server.servlet.path}/error,\
+  ${server.servlet.path}/swagger-ui/**,${server.servlet.path}/v3/api-docs/**,/v1/notifier/actuator/prometheus,\
+  ${server.servlet.path}/system-info/**
+
+
+# Authentication ignored endpoint (permit-all)
+mosip.idp.security.ignore-auth-urls=${server.servlet.path}/csrf/**,${server.servlet.path}/authorization/**,\
+  ${server.servlet.path}/linked-authorization/**,${server.servlet.path}/oidc/**,${server.servlet.path}/oauth/**,\
+  ${server.servlet.path}/actuator/**,/favicon.ico,${server.servlet.path}/error,${server.servlet.path}/swagger-ui/**,\
+  ${server.servlet.path}/v3/api-docs/**,/v1/notifier/actuator/prometheus
+
+mosip.idp.amr-acr-mapping-file-url=https://raw.githubusercontent.com/mosip/mosip-config/qa-1201-B2/amr-acr-mapping.json
```

Comment:
This URL should be a placeholder

Reply:
plz use the below value instead of providing GitHub URL

`${spring_config_url_env}/*/${active_profile_env}/${spring_config_label_env}/amr-acr-mapping.json`